pkg/rpm/resolver/resolver.go

```diff
@@ -72,6 +77,16 @@ func New(workDir string, podman Podman, baseImageBuilder BaseResolverImageBuilde
 func (r *Resolver) Resolve(packages *image.Packages, localRPMConfig *image.LocalRPMConfig, outputDir string) (rpmDirPath string, pkgList []string, err error) {
 	zap.L().Info("Resolving package dependencies...")
 
+	revert, err := mount.DisableDefaultMounts(r.overrideMountsPath)
+	if err != nil {
+		return "", nil, fmt.Errorf("temporary disabling automatic volume mounts: %w", err)
+	}
+	defer func() {
+		if err = revert(); err != nil {
```

Comment:
This will modify the returned `err` value. We should use separate variable for it.